gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -1038,6 +1038,12 @@
   public static final String AVRO_SCHEMA_CHECK_STRATEGY_DEFAULT =
       "org.apache.gobblin.util.schema_check.AvroSchemaCheckDefaultStrategy";
 
+  /**
+   * Configuration and constant vale for GobblinMetadataChangeEvent
+   */
+  public static final String LIST_DELIMITER_KEY = ",";
```

Comment:
Does it make more sense to put theses into `GobblinMetadataChangeEvent` itself?

Reply:
This value will also be used by extractor, which will set the offset range of the data topic. So to decouple the extractor and gmce publisher, I put it in the common place